app/models/test_track/offline_session.rb

```diff
@@ -12,6 +12,10 @@ def self.with_visitor_for(identifier_type, identifier_value)
     end
   end
 
+  def self.create_visitor_for(identifier_type, identifier_value)
+    new(identifier_type, identifier_value).send :notify_unsynced_assignments!
```

Comment:
Regardless, this method's behavior is probably moot because I think the caller either wants to blow up or noop instead of calling this method.